sources/packages/api/src/database/migrations/1628282476832-CoeStatusType.ts

```diff
@@ -0,0 +1,12 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { getSQLFileData } from "../../utilities";
+
+export class CoeStatusType1628282476832 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(getSQLFileData("Create-CoeStatus.sql", "Types"));
```

Comment:
I think, we agreed that .sql name will be `hyphen-case`

Reply:
Please adjust all the .sql files as per @ann-aot comments.